builder/index.html

```diff
@@ -44,7 +44,6 @@ <h3 class="modal-title">Unsaved changes</h3>
             <button class="btn btn-default" ng-disabled="editForm.$invalid" ng-click="save()">Save Changes</button>
           </div>
         </script>
-        </script>
```

Comment:
shouldn't we xml validate (or whatever we could use) to prevent errors or mismatch in opening/closing tags ?